indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -419,6 +419,9 @@ def get_track_reviewing_state(self, track):
         else:
             return AbstractReviewingState.mixed
 
+    def get_track_reviews(self, track):
```

Comment:
Well, I need it for the PDF generation. And I don't think there's a place more appropriate than the model.